src/CMakeLists.txt

```diff
@@ -1,549 +1,534 @@
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/hashmgr.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/htypes.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/hunspell.cxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/hunspell.h
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/hunspell.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/hunvisapi.h
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/hunvisapi.h.in
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/hunzip.cxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/hunzip.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/langnum.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/phonet.cxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/phonet.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/replist.cxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/replist.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/suggestmgr.cxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/suggestmgr.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/utf_info.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/hunspell/w_char.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/parsers/firstparser.cxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/parsers/firstparser.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/parsers/htmlparser.cxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/parsers/htmlparser.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/parsers/latexparser.cxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/parsers/latexparser.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/parsers/manparser.cxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/parsers/manparser.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/parsers/odfparser.cxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/parsers/odfparser.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/parsers/textparser.cxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/parsers/textparser.hxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/parsers/xmlparser.cxx
+    libraries/sonnet/src/plugins/hunspell/hunspell/src/parsers/xmlparser.hxx
+)
+endif()
+
+if (APPLE)
+   list(APPEND SOURCE_FILES
+    libraries/sonnet/src/plugins/nsspellchecker/nsspellcheckerclient.h
+    libraries/sonnet/src/plugins/nsspellchecker/nsspellcheckerclient.mm
+    libraries/sonnet/src/plugins/nsspellchecker/nsspellcheckerdebug.cpp
+    libraries/sonnet/src/plugins/nsspellchecker/nsspellcheckerdebug.h
+    libraries/sonnet/src/plugins/nsspellchecker/nsspellcheckerdict.h
+    libraries/sonnet/src/plugins/nsspellchecker/nsspellcheckerdict.mm)
+endif()

```

Comment:
`nsspellchecker` is disabled, this part can safely be removed.

Reply:
So I can fully remove it? 
And only keep hunspell?

Reply:
yes,
but the correct way  to handle this would be to build `sonnet` as a library (hunspell will be a dependency there) and link that statically into QON

Reply:
I should mention here that KDE provides Cmake files for sonnet however they depend on ECM(Extra cmake modules) to work correctly. So thats not an option

Reply:
> yes,
> but the correct way to handle this would be to build `sonnet` as a library (hunspell will be a dependency there) and link that statically into QON

Ok. In time.